src/server/ocpp/services/OCPPService.ts

```diff
@@ -984,6 +1003,8 @@ export default class OCPPService {
     await this.notifyStatusNotification(tenantID, chargingStation, statusNotification);
     // Send new status to IOP
     await this.updateOCPIStatus(tenantID, chargingStation, foundConnector);
+    // Send new status to Hubject
+    await this.updateOICPStatus(tenantID, chargingStation, foundConnector);
```

Comment:
Since you seem to have the transaction, you know which function to call ... so do that check everywhere.

Reply:
What check do you mean? I only added the update of the OICP Status.

Reply:
The branching that will call either OCPI or OICP depending code.

Reply:
ah okay. but for the update of the connector status, the status should be updated for both OCPI and OICP.

Reply:
Only if the component is enable in the tenant, if the station is public, etc. So you can also check if the status need to be updated before the call (it's clearer than inside).

Reply:
okay thank you. I changed it